test/test_utils.go

```diff
@@ -281,3 +287,33 @@ func verifySignature(ctx context.Context, t *testing.T, c *clients, obj objects.
 		}
 	}
 }
+
+// restartChainsControllerPod restarts the pod running Chains
+// it then waits for a given timeout for the pod to resume running state
+func restartChainsControllerPod(ctx context.Context, c kubernetes.Interface, timeout time.Duration) error {
+	pods, err := c.CoreV1().Pods("tekton-chains").List(ctx, metav1.ListOptions{LabelSelector: "app.kubernetes.io/component=controller"})
+	if err != nil {
+		return err
+	}
+	pod := pods.Items[0]
+	podUid := pod.UID
+	gradePeriodSeconds := int64(0)
```

Comment:
```suggestion
	gracePeriodSeconds := int64(0)
```

Not sure if this is a nit change 😄

Reply:
Fixed 😄